c7n/resources/ec2.py

```diff
@@ -384,6 +385,37 @@ def process_instance(i):
 
         with self.executor_factory(max_workers=2) as w:
             list(w.map(process_instance, instances))
+
+@actions.register('create-snapshot')
```

Comment:
i think its a bit nicer to just call this action snapshot, actions already imply verbs.